vald/templates/agent/daemonset.yaml

```diff
@@ -20,6 +20,8 @@ metadata:
   name: {{ .Values.agent.name }}
   labels:
     app: {{ .Values.agent.name }}
+  annotations:
+    checksum/configmap: {{ include (print $.Template.BasePath "/agent/configmap.yaml") . | sha256sum }}
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
it is required to discuss about the agents should have sha256 checksum or not.

Reply:
for now agent does not have their own scheduler, if we change configmap agent pod will be restart it means agent index will be flush.
we should disable this annotation until we impl agent scheduler

Reply:
okay. i've just commented out these annotations.